Application/src/DiagramCheck.java

```diff
@@ -0,0 +1,49 @@
+import javafx.scene.control.Alert;
+import java.util.Collection;
+
+/**
+ * @version 0.1
+ * @author Tim Jonasson & Kosara Golemshinska
+ * @since 2017-10-10
+ */
+
+public class DiagramCheck {
+	// Supported diagram types.
+	public static String diagrams = "sequence_diagram\n";
```

Comment:
Isn't this redundant since you have 'allowedDiagrams' furher down?

Reply:
Sssshhhhh